evm-contracts/src/v0.7/dev/OperatorGenerator.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.7.0;
+
+import "./Operator.sol";
+
+contract OperatorGenerator {
```

Comment:
Named Generator because of issues with ethers. It didn't like `OperatorFactoryFactory`.

Reply:
What about OperatorContractFactory?

Reply:
Factory is the name of the pattern, I think it's very clear to people familiar with it. I wouldn't optimize it for the tests.

Maybe alias it when you import in the tests instead?
```
import { OperatorFactoryFactory as OperatorGeneratorFactory } from '../../ethers/v0.7/OperatorFactoryFactory'
```

Reply:
I get that, but... it looks like an OperatorFactory, i.e. a Factory that produces Operators, why would you repeat Factory? I don't think it's outputting OperatorFactories is it?

Reply:
Sorry, this needs more context. When ethers compiles the contracts for testing, it creates "Factory" classes for each contract.

For example, with the `Operator` contract, ethers generates an `Operator.d.ts` file and an `OperatorFactory.ts` file. This factory class is used in our tests to deploy Operators and test them. So we're not appending the extra `Factory` intentionally @j16r, ethers does it when it compiles the contracts.

The issue here is that by also having an `OperatorFactory` contract, ethers generates an `OperatorFactory.d.ts` file and an `OperatorFactoryFactory.ts` file.

<img width="321" alt="Screenshot 2020-11-18 at 09 56 52" src="https://user-images.githubusercontent.com/6523673/99515597-1256ce00-2985-11eb-8f49-9a125227bc50.png">

So, this should be fine, but it isn't. Ethers then has problems with importing the correct `OperatorFactory` file in the `OperatorFactoryFactory`. This is the problem line:

```
import { OperatorFactory } from "./OperatorFactory";
```

This should import `OperatorFactory.d.ts`, but it doesn't, it imports `OperatorFactory.ts`.

Reply:
Looks like it's an issue with TypeChain: https://github.com/ethereum-ts/TypeChain/issues/225

Reply:
I've renamed this contract to `ChainlinkOperatorFactory` for now. Played around with upgrading TypeChain in belt and evm-test-helpers to no avail. Maybe @se3000 could provide more insight there.

In the meantime, this naming works.